scripts/set_kokkos_env.sh

```diff
@@ -0,0 +1,12 @@
+# set nvcc as the underlying C++ compiler for Open MPI
+export OMPI_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set nvcc as the underlying C++ compiler for MPICH
+export MPICH_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set the original C++ compiler as the underlying compiler for the nvcc
+# wrapper
+export NVCC_WRAPPER_DEFAULT_COMPILER=$PATH_TO_CXX_COMPILER
+
+# set all cuda calls to blocking launch
+export CUDA_LAUNCH_BLOCKING=1
```

Comment:
I'm not sure I understand. Tpetra is not optional in DTK, so you need to make sure Tpetra is being run correctly. For that, you need CUDA_LAUNCH_BLOCKING, and there is not avoiding that. You `fence()` call may fix this isolated unit test but apps will still need the env variable.